@navikt/core/react/src/stepper/Stepper.tsx

```diff
@@ -11,6 +11,10 @@ export interface StepperProps extends React.HTMLAttributes<HTMLOListElement> {
    * Adds classname to wrapper
    */
   className?: string;
+  /**
+   * The direction the component grows. Default is "vertical".
+   */
+  horisontal?: boolean;
```

Comment:
Jeg foretrekker navngitte argumenter her, på samme måte som material-ui har gjort: https://mui.com/material-ui/api/stepper/#props
```suggestion
  /**
   * The direction the component grows. 
   * @default"vertical"
   */
orientation: "horizontal" | "vertical";
```